randovania/games/samus_returns/presets/multiworld-starter-preset.rdvpreset

```diff
@@ -0,0 +1,285 @@
+                        1
+                    ],
+                    "pickup_count": 30,
+                    "requires_main_item": true
+                },
+                "Power Bomb Tank": {
+                    "ammo_count": [
+                        1
+                    ],
+                    "pickup_count": 15,
+                    "requires_main_item": true
+                },
+                "Aeion Tank": {
+                    "ammo_count": [
+                        50
+                    ],
+                    "pickup_count": 15,
+                    "requires_main_item": true
+                }
+            }
+        },
+        "damage_strictness": 1.5,
+        "pickup_model_style": "all-visible",
+        "pickup_model_data_source": "etm",
+        "logical_resource_action": "randomly",
+        "first_progression_must_be_local": false,
+        "minimum_available_locations_for_hint_placement": 5,
+        "minimum_location_weight_for_hint_placement": 0.2,
+        "dock_rando": {
+            "mode": "vanilla",
+            "types_state": {
+                "door": {
+                    "can_change_from": [
+                        "Charge Beam Door",
+                        "Missile Door",
+                        "Plasma Beam Door",
+                        "Power Beam Door",
+                        "Power Bomb Door",
+                        "Spazer Beam Door",
+                        "Super Missile Door",
+                        "Wave Beam Door"
+                    ],
+                    "can_change_to": [
+                        "Charge Beam Door",
+                        "Missile Door",
+                        "Plasma Beam Door",
+                        "Power Beam Door",
+                        "Power Bomb Door",
+                        "Spazer Beam Door",
+                        "Super Missile Door",
+                        "Wave Beam Door"
+                    ]
+                }
+            }
+        },
+        "single_set_for_pickups_that_solve": true,
+        "staggered_multi_pickup_placement": true,
+        "check_if_beatable_after_base_patches": false,
+        "teleporters": {
+            "mode": "vanilla",
+            "excluded_teleporters": [],
+            "excluded_targets": []
+        },
+        "energy_per_tank": 100,
+        "starting_energy": 99,
+        "starting_aeion": 1000,
+        "life_tank_size": 299,
+        "aeion_tank_size": 500,
+        "missile_tank_size": 30,
+        "super_missile_tank_size": 10,
+        "elevator_grapple_blocks": false,
+        "area3_interior_shortcut_no_grapple": true,
+        "charge_door_buff": true,
+        "beam_door_buff": true,
+        "beam_burst_buff": true,
+        "nerf_super_missiles": false,
+        "surface_crumbles": true,
+        "area1_crumbles": true,
+        "reverse_area8": false,
+        "allow_highly_dangerous_logic": false,
+        "artifacts": {
+            "prefer_metroids": true,
```

Comment:
Still unsure if we should use DNA anywhere as standard or not. Opinions?

Reply:
feel like dna anywhere might be nicer.

Reply:
DNA Anywhere makes for a more interesting experience which I feel ties into multi a lot better.